lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -468,25 +496,31 @@ class DecomposeAtenLogSoftmaxIntOp
     if (!tensorType.hasDtype() || !tensorType.getDtype().isa<mlir::FloatType>())
       return rewriter.notifyMatchFailure(op, "Only support floating type");
 
-    // softmax(x, dim)
-    Value softmax = rewriter.create<AtenSoftmaxIntOp>(loc, tensorType, self,
-                                                      dim, op.dtype());
-    rewriter.replaceOpWithNewOp<AtenLogOp>(op, op.getType(), softmax);
+    Value logSoftmax = getLogSoftmaxResult(op, rewriter);
```

Comment:
A check is needed to make sure the return of `getLogSoftmaxResult` is not `nullptr`. The same applies for the `Aten_LogSoftmaxOp` version

Reply:
Done, thanks for the catch.